tasks/deploy.py

```diff
@@ -359,50 +368,51 @@ def upload_tarball(job_dir, build_target, timestamp, repo_name, pr_number, pr_co
     upload_cmd = ' '.join(cmd_args)
 
     # run_cmd does all the logging we might need
-    out, err, ec = run_cmd(upload_cmd, 'Upload tarball to S3 bucket', raise_on_error=False)
+    out, err, ec = run_cmd(upload_cmd, 'Upload artefact to S3 bucket', raise_on_error=False)
 
     if ec == 0:
         # add file to 'job_dir/../uploaded.txt'
-        append_tarball_to_upload_log(tarball, job_dir)
+        append_tarball_to_upload_log(artefact, job_dir)
```

Comment:
Should this function be renamed so that tarball is not used anymore and it is more consistent?

Reply:
Implemented in https://github.com/EESSI/eessi-bot-software-layer/pull/263/commits/e705494722c82c773d6df63476c2b4ceef8a0292